insights/specs/sos_archive.py

```diff
@@ -46,6 +46,7 @@ class SosSpecs(Specs):
     hostname = first_of([simple_file("sos_commands/general/hostname_-f"), simple_file("sos_commands/general/hostname")])
     httpd_conf_sos = glob_file(["/conf/httpd/conf/httpd.conf", "/conf/httpd/conf.d/*.conf"])
     installed_rpms = simple_file("installed-rpms")
+    init_process_cgroup = simple_file("/proc/1/cgroup")
```

Comment:
Since the full-name in sos is the same to it in `default`, this definition in sos_archive can also be omitted.